scripts/build-windows.sh

```diff
@@ -4,8 +4,10 @@
 #!/bin/bash
 EXE_NAME="halloy.exe"
 TARGET="x86_64-pc-windows-msvc"
+VERSION=$(cat VERSION).0
```

Comment:
We should keep `Product version` and `File version` equal imho.
And preferably both stick to our normal schematics.

Reply:
Cannot remove the trailing 0. If i removing it, then `cargo-set-version set-version $VERSION` is failing.